hazelcast/src/main/java/com/hazelcast/config/alto/AltoSocketConfig.java

```diff
@@ -1,58 +1,86 @@
 package com.hazelcast.config.alto;
 
-import com.hazelcast.config.InvalidConfigurationException;
+import com.hazelcast.spi.annotation.Beta;
 
 import javax.annotation.Nonnull;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
+import static com.hazelcast.internal.util.Preconditions.checkNotNull;
+import static com.hazelcast.internal.util.Preconditions.checkPositive;
+
+/**
+ * Socket configuration for Alto. In Alto, each eventloop has its own
```

Comment:
This is not completely correct.

Each eventloop can have many sockets.

First of all it can have many server sockets. Maybe a socket for WAN, another one for clients and another one for members.

And each eventloop could process many many non server sockets (sometimes called client sockets.. but should not be confused with HZ clients).

Reply:
Documentation needs some work.

Reply:
This is stale state. Please see the current doc from the files tab.

Reply:
I do realize that current doc is short. If you have a better explanation, I am happy to change.
```
/**
 * Socket configuration for Alto. In Alto, each eventloop has its own
 * sockets.
 *
 * @see AltoConfig
 * @since 5.3
 */
```

Reply:
Basic but not incorrect.